makefiles/pseudomodules.inc.mk

```diff
@@ -9,6 +9,7 @@ PSEUDOMODULES += conn_can_isotp_multi
 PSEUDOMODULES += cord_ep_standalone
 PSEUDOMODULES += core_%
 PSEUDOMODULES += cortexm_fpu
+PSEUDOMODULES += cortexm_svc
```

Comment:
This should be provided in `cpu/cortexm_common/Makefile.features`

Reply:
yup